doc/source/release/1.17.3-notes.rst

```diff
@@ -15,7 +15,7 @@ OpenBLAS >= 3.7 to avoid errors on the Skylake architecture.
 Highlights
 ==========
 
-- Wheels for Python 3.8
+- Wheels for Python 3.9
```

Comment:
I don't think this is correct: 1.17 did not release wheels for 3.9, right?

Reply:
Correct, 3.9 is still up in the air. For 1.17, 3.8 is the latest.